benchmark/benchmarks.jl

```diff
@@ -31,8 +31,8 @@ energy_problem = EnergyProblem(connection)
 
 SUITE["energy_problem"]["create_model"] = @benchmarkable begin
     create_model!($energy_problem)
-end samples = 3 evals = 1 seconds = 86400
-create_model!(energy_problem)
+end samples = 2 evals = 1 seconds = 86400
```

Comment:
```suggestion
end samples = 3 evals = 1 seconds = 86400
```
Since it's not running, might as well use the default.